pages/people_page.py

```diff
@@ -0,0 +1,57 @@
+from selenium.webdriver.common.keys import Keys
+from .base_page import Page
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+import random
+import string
+
+
+class PeoplePage(Page):
+
+    url = '/people'
+
+    def show_admins_only(self):
+        self.context.driver.find_element_by_id('show_admins_only').click()
+        admin = self.context.wait.until(EC.visibility_of_element_located((By.XPATH, '//a[contains(text(), "Tim Bortnik")]')))
+        if admin:
+            print('\tAdmins found correctly')
+            pass
+        else:
+            print("Admins not found")
+
+    def show_all_users(self):
+        self.context.driver.find_element_by_id('show_all_users').click()
+        self.context.wait.until(EC.visibility_of_element_located((By.XPATH, '//ol[@class="aui-nav-pagination"]')))
+
+    def we_see(self):
```

Comment:
From method name I don't understand what it gonna doing, please provide more correct name.